selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -1446,6 +1484,11 @@ public void waitSuccessCondition(ExpectedCondition<Boolean> expression) {
     waitSuccessCondition(expression, DEFAULT_TIMEOUT);
   }
 
+  /** Hides context menu. */
+  public void hideContextMenu() {
+    actionsFactory.createAction(seleniumWebDriver).moveByOffset(-1, -1).click().build().perform();
```

Comment:
getAction().moveByOffset(-1, -1).click().perform();

Reply:
Sorry, I don't understand what you mean.